### ISSUE TITLE
insert_as_prev_sibling

### ISSUE THREAD

**Issue by miket3** [evopix-archive/orm-mptt#10]
I dont know what the fix is yet.  But insert_as_prev_sibling()   doesnt work.    

1st it calls parent_from()  which also returns the wrong id.  It returns the same id it was sent instead of the parent_id.

Then i would imagine that it is not sending the correct params to insert() or insert() is doing something wrong as well.

off top of my head i think what needs to be done is:
1. fix parent_from.   I just returned $this->parent_id instead of $target. to fix it for me.
2. then call insert() using the $target.  not the parent_from result.
3. take those results and just replace the parent_id with parent_from() $target.

I would imagine that the insert_as_next_sibling is same thing.

**Comment by miket3** [comment]
here is my quick fix.

```
public function insert_as_prev_sibling($target)
{
    $parent_id = $this->parent_from($target, 'parent_id');
    $this->insert($target, $this->left_column, 0, 0);
    $this->parent_id = $parent_id;
    parent::save();
    return $this;
}
```

**Comment by miket3** [comment]
my quick fix doesnt work either.  so this function still does not work.

**Comment by evopix** [comment]
parent_id is now properly set in insert_as_prev_sibling and insert_as_next_sibling methods. Closed by a0c4760b853768e7ccdbccda03e541b1a07b63b3.
